dotCMS/src/main/java/com/dotcms/tika/TikaUtils.java

```diff
@@ -158,6 +161,23 @@
                     "Could not parse file metadata for file : " + binFile.getAbsolutePath() + ". " + ex.getMessage());
             }
         }
+        User systemUser = null;
+        try {
+            systemUser = APILocator.getUserAPI().getSystemUser();
+        } catch (DotDataException e) {
+            Logger.error(this,"Unable to get System User. ",e);
+        }
+
+        Map<String, Object> additionProps = new HashMap<>();
```

Comment:
not sure if give memory and then override is good, I would set to Collections.emptyMap() on the catch block and set it to null at the begin